### DISCUSSION TITLE
用赞助版本一直被防毒杀掉怎么办...

### DISCUSSION THREAD

**Discussion by snowolf50161** [Meloong-Git/PCL#4467] [提问] [answered]
每隔一段时间赞助版本就被防毒杀掉...我用的是PC-Cillin防毒
都已经加入例外清单了还是一直被杀掉...
![螢幕擷取畫面 2024-08-10 233548](https://github.com/user-attachments/assets/b2deaf1e-8187-4724-afb6-b23a0af2b97d)

**Comment by allMagicNB** [comment]
#666

**Comment by Ad-closeNN** [comment]
为什么不用火绒(

**Reply by snowolf50161** [reply]
PC-Cillin對我來說比較友善

**Comment by MoYuan-CN** [comment]
請參見 #56 #666，被防毒軟體殺掉是因爲 PCL 有對程序進行保護加殼
請參考您使用的防毒軟體的指南，將 PCL 加入其白名單中

**Reply by snowolf50161** [reply]
我試過加入白名單
但是還是被殺掉了（正式版沒有這個問題不知道為什麼）

**Reply by MoYuan-CN** [reply]
因爲只有快照版加了殼，加過白名單但還是殺掉了的話……
1. 找一下殺毒軟體的支持團隊
2. 換一個殺毒軟體

**Reply by snowolf50161** [reply]
我已傳送殺毒軟體的支持團隊說這件事了，也許過一陣子會改善

**Reply by snowolf50161** [reply]
跟防毒的客服連絡過了，防毒的客服人員(和團隊）無法處理這件事

**Comment by lactobionicAcid** [comment]
應該是 Microsoft Defender SmartScreen 在後臺偷偷把 PCL 殺掉了...

簡單的解決辦法：
開啟 “Windows 安全性”，轉到 “應用程式與瀏覽器控制”，點選 “信譽評等防護設定”，然後把 “檢查應用程式和檔案”選項關掉。

**Reply by snowolf50161** [reply]
這個嗎？
![螢幕擷取畫面 2024-08-13 012639](https://github.com/user-attachments/assets/b8816ec5-5745-4149-9d77-2125ed7fd47c)

**Reply by snowolf50161** [reply]
不行...還是被防毒殺掉了

**Comment by WForst-Breeze** [comment, marked as answer]
此类情况大部分是由于 Microsoft Defender 造成的（#666）

先尝试将 PCL2  加入 Microsoft Defender 排除项（可上网搜一下具体步骤，或者如果可以的话，PCL2 内置的帮助库中有详细教程）中，如若问题仍未解决请检查是否正确将 PCL2 加入杀毒软件白名单中。

在进行完第一步后，至少 Microsoft Defender 就 **大概率** 不会再把 PCL 静默删除了，如果仍不能解决那可能就是杀毒软件的问题了 😢

**Reply by snowolf50161** [reply]
![image](https://github.com/user-attachments/assets/9bc29379-1814-46e0-8d21-3455a7541f3e)
目前看來真的是 #666 上出現的問題，應該還原就好了
被認定為trojan 病毒（特洛伊病毒）了

**Comment by wuliaodexiaoluo** [comment]
實在不行建議看下殺軟有沒有文件防護選項，給 PCL 加入防護列表，這樣 wd 刪文件的時候會被主防攔截（可能會導致更新異常?）

**Reply by snowolf50161** [reply]
剛剛加了，是Windows自帶的防毒擋下來的

**Comment by zzr-shift** [comment]
其实还有一种方法……
来一个有钱人给PCL2买一个证书（应该是这样吧）

**Reply by zzr-shift** [reply]
额 数字签名

**Reply by MoYuan-CN** [reply]
现在的正式版好像是有签名的？

**Reply by zzr-shift** [reply]
好像还没有罢（悲）

**Reply by MoYuan-CN** [reply]
![image](https://github.com/user-attachments/assets/aea9797f-396e-4693-baaa-43be572dc549)
很不巧的是，这是 PCL Release。PCL Snapshot 并没有签名

**Reply by zzr-shift** [reply]
为了照顾大部分玩家吧 不然大家下载一次消失一次有点太…

**Reply by allMagicNB** [reply]
> 为了照顾大部分玩家吧 不然大家下载一次消失一次有点太…

这是快照版特性。
